### PR TITLE
JDK-8283015: Create a test for JDK-4715496

### PR DESCRIPTION
**Pull request by srmandal**
Create a test for JDK-4715496

Issue observed in the above bug is that the JTable.AccessibleJTableCell.getAccessible name returns the cell instance string instead of the cell text.
The test is being added to verify the same.
This review is for migrating tests from a closed test suite to open.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8283015](https://bugs.openjdk.java.net/browse/JDK-8283015): Create a test for JDK-4715496

### Reviewers
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7787/head:pull/7787` \
`$ git checkout pull/7787`

Update a local copy of the PR: \
`$ git checkout pull/7787` \
`$ git pull https://git.openjdk.java.net/jdk pull/7787/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7787`

View PR using the GUI difftool: \
`$ git pr show -t 7787`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7787.diff">https://git.openjdk.java.net/jdk/pull/7787.diff</a>

</details>

### CODE REVIEW

File: test/jdk/javax/accessibility/8283015/AccessibleJTableCellNameTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @key headful
+ * @bug 4715496
+ * @summary AccessibleJTableCell.getAccessible name incorrectly returns cell instance string instead of cell text.
```
Comment (mrserb) [inline test/jdk/javax/accessibility/8283015/AccessibleJTableCellNameTest.java:28]: Please split the long lines, use the 80 chars per line.

File: test/jdk/javax/accessibility/8283015/AccessibleJTableCellNameTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    private static void doTest() throws Exception {
+        try {
+            SwingUtilities.invokeAndWait(() -> createGUI());
+            Robot robot = new Robot();
+            robot.setAutoDelay(500);
+            robot.waitForIdle();
+
+            for (int i = 0; i <= colNames.length - 1; i++) {
+                Accessible accessible = jTable.getAccessibleContext().getAccessibleTable()
```
Comment (mrserb) [inline test/jdk/javax/accessibility/8283015/AccessibleJTableCellNameTest.java:60]: I remember a similar code in some other test which probably was integrated already. But I just found that the jtable here is a Swing component that should be accessed in the EDT.

**Comment by srmandal** [comment]
The test ran successfully on Mach5 with multiple runs (30) on windows-x64, linux-x64 and macos-x64.

**Review by mrserb — APPROVED** [review]

**Comment by srmandal** [comment]
@mrserb Can you please sponsor this PR ?

**Pull request closed without merging**